src/Management/test/Endpoint.Test/Actuators/HttpExchanges/HttpExchangesDiagnosticObserverTest.cs

```diff
@@ -212,7 +212,7 @@ public void GetHttpExchanges_FiltersDuringReturn()
 
         optionsMonitor.CurrentValue.RequestHeaders.Add(HeaderNames.Authorization);
         optionsMonitor.CurrentValue.RequestHeaders.Add("header1");
-        optionsMonitor.CurrentValue.ResponseHeaders.Add("headera");
```

Comment:
The casing was intentionally wrong here to validate case-insensitive matching. I don't know if that's a critical loss though, what do you think?

Reply:
I wasn't aware of that, but it's a good thing to test for. I've reverted the change and added a comment to clarify the intent.